doc/release/upcoming_changes/14051.expired.txt

```diff
@@ -0,0 +1,2 @@
+The deprecation of ``expand_dims`` out-of-range axes in 1.13.0 has
```

Comment:
Can you make this a bullet point, indenting the second line? Most are right now. (I am still hoping towncrier will add reasonable support for the formatting issues, since they did add multiple file support, but...)

Reply:
Fixed, thanks!

Reply:
Ack, sorry I missed it before. But I think towncrier only picks up `.rst` and no suffix at all, so could you change `.txt` to `.rst`? (not quite sure immediately if this is otherwise finished)

Reply:
Sorry I missed that too.  It's fixed now.